bftengine/src/bftengine/FullNodeReplica.hpp

```diff
@@ -0,0 +1,81 @@
+// Concord
+//
+// Copyright (c) 2018-2019 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").  You may not use this product except in
+// compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license terms. Your use of
+// these subcomponents is subject to the terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "ReplicaForStateTransfer.hpp"
+#include "util/Timers.hpp"
+#include "CheckpointInfo.hpp"
+
+namespace bftEngine::impl {
+
+class ClientRequestMsg;
+/**
```

Comment:
Did you plan to write here some explenation about the class?
If not yet (I think you should keeo it empoty) - just add a TODO. The way it is now is unclear.

Reply:
will add a comment  in the beginning of the file and in the PR description i will mention about the same